client/src/components/GameWindow/Game.js

```diff
@@ -48,6 +49,7 @@ const GameWindow = ({ room, name, socket, closeGame}) => {
 		<div className="container">
 			<InfoBar room={room} onClick={closeGame}/>
 			<Messages messages={messages} name={name} />
+			<ProgressBar />
```

Comment:
we might need to think about what props the progress bar is receiving, or under what conditions it will be displayed, but that can be in the next story.

Reply:
Exactly. At the moment it receives only one prop. But this would all depend on the parent container logic